src/fitnesse/wiki/fs/PageXmlizer.java

```diff
@@ -43,7 +43,7 @@ public void deXmlizeSkippingRootLevel(Document document, WikiPage context, Xmliz
   public Document xmlize(PageData data) {
     Document document = XmlUtil.newDocument();
     Element dataElement = document.createElement("data");
-    XmlUtil.addCdataNode(dataElement, "content", data.getContent());
+    XmlUtil.addTextNode(dataElement, "content", data.getContent());
```

Comment:
Why is this no longer a 'cdata' block? This probably means the content will be xml escaped, which it was not before. Will that not make it harder to edit the content as 'plain text'? Can it not remain cdatan for maximum backward compatibility?

Reply:
In Java 9, the default XML transformation will put a CDATA section on a new line. When parsing the content, the parsed text does include extra newlines and spacing. This is not what we want.
Just using text avoids this.

This part is only used for the SerializationResponder (the proxy endpoint). Nowhere else. The page history is recorded as normal escaped text already and that's been working for years already.